File: TenMinsPy/day39.py
# ask for date input and convert to unix timestamp

from datetime import datetime, timezone

class day39:
    def __init__(self):
        nd = input("Enter a date in YYYY-MM-DD format\n")
        days = nd.split("-") #this is an array of sorts
        dt = datetime(int(days[0]), int(days[1]), int(days[2]))  # 2025-11-28 14:30
        unix_ts = int(dt.replace(tzinfo=timezone.utc).timestamp())
        print(unix_ts)

day39()

File: TenMinsJava/day39.java
// ask for date input and convert to unix timestamp
package TenMinsJava;

import java.time.LocalDate;
import java.util.Scanner;

public class day39 {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        LocalDate d = LocalDate.now();
        long epoch = d.toEpochDay() * 24 * 60 * 60;
        System.out.println("Enter a date in format YYYY-MM-DD");
        String nd = scan.nextLine();
        LocalDate inputDate = LocalDate.parse(nd);
        long nEpoch = inputDate.toEpochDay() * 24 * 60 * 60;
        System.out.println(nEpoch - epoch);
    }
}
